README.md

```diff
@@ -121,8 +120,8 @@ Content-Type" application/javascript
 The package in this case contains a lot of pages with resources ("Encyclopedia in a file") or multiple sites (in subpackages). The proposed structure facilitates efficient access, assuming the whole package is available locally. Several important notes:
 
 1. The Link: header with **rel=index** declares a specified part to be a Content Index of the package. The **offset=12014** attribute specifies the octet offset/size from the beginning of the Package Header of the package to the Content Index part. That can be used in file-seek APIs to quickly read the part without a need to parse the potentially huge package itself.
-2. Content Index part is typically generated during package creation, it doesn't have a natural URL. We propose to use cid: generated URLs (UUID-based, 128-bit) for such generated parts. The visibility scope of those URLs is limited similar to package boundaries, and is for the current package only.
-3. Content-type of the Content Index is application/index.
+2. Content Index part is typically generated during package creation, it doesn't have a natural URL. We propose to use [`cid:`](https://tools.ietf.org/html/rfc2392) generated URLs (UUID-based, 128-bit) for such generated parts. The visibility scope of those URLs is limited similar to package boundaries, and is for the current package only.
```

Comment:
Right. The cid: schema is not really the right one here. Chrome uses URLs like cid:[uuid] in MHTML generation and this is apparently not standard, but then again MHTML is not well formed standard either... I've picked the cid: schema after Chrome's MHTML implementation, but perhaps urn:uuid: would work better, and it is standard:  http://www.ietf.org/rfc/rfc4122.txt
If you think it is better here, please feel free to change to it or let me know and I'll change after merging this PR.